pkgs/desktops/mate/default.nix

```diff
@@ -42,6 +42,7 @@ let
     mate-sensors-applet = callPackage ./mate-sensors-applet { };
     mate-session-manager = callPackage ./mate-session-manager { };
     mate-settings-daemon = callPackage ./mate-settings-daemon { };
+    mate-settings-daemon-wrapped = callPackage ./mate-settings-daemon/wrapped.nix { };
```

Comment:
Why not the more common naming convention of `thing` and `thing-unwrapped`?

Reply:
Well, the -wrapped one is double-wrapped adding only mate-control-center that would cause infinite recursion otherwise